src/components/dialogs/parameters/dynamicsimulation/curve-parameters.tsx

```diff
@@ -137,7 +137,7 @@ const CurveParameters = ({ path }: { path: string }) => {
 
     return (
         <>
-            <Grid container direction={'column'} sx={{ height: 640 }}>
```

Comment:
Why this value in pixels ? why not 100% ? why not flexGrow 1 ?

Reply:
It is a quick fix when we add horizontal bar on top of all tabs, it is not really related this PR.. I will fix later, not only for DS but also others tables